src/db/MariadbIfc.py

```diff
@@ -400,14 +399,14 @@ def getProfile(self,
 
         return profile
 
-    def getProfiles(self,
+    def getUsersProfiles(self,
                     user_id : int) -> list:
```

Comment:
nit: vertical alignment is off for the last 2 parameters (maybe a tab snuck in somewhere)?